### DISCUSSION TITLE
What does bs mean?

### DISCUSSION THREAD

**Discussion by kishaningithub** [brianvoe/gofakeit#164] [Q&A] [answered]
I would like to what the abbreviation BS() means?

From the code i see it referers to all these https://github.com/brianvoe/gofakeit/blob/425ab1f3321604172bd6688b6879b7a5f08947f2/data/company.go#L8

But would like to know what "BS" actually stands for?

**Comment by brianvoe** [comment, marked as answer]
Its a pretty common usage of company fluff words that dont really mean anything. Youll see it in most fake data generators in various languages. (It means bull shit)

**Comment by brianvoe** [comment]
Descriptions are being update soon as well.
